frontend/app/types/index.ts

```
@@ -21,4 +21,73 @@ export interface Workspace {
+    workspace:Workspace;
+    startDate:Date;
+    dueDate:Date;
+    progress:number;
+    tasks:Task[];
+    members:{
+        user:User;
+        role:"admin"|"member"|"owner"|"viewer"
+    }[];
+    createdAt:Date;
+    updatedAt:Date;
+    isArchived:Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)